### PR TITLE
common/hooks: Prefill update info in commit msg

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**
- If the version has been changed in the package.yml as part of the commit then prefill that information in the commit header. e.g. nano: Update to 1.2.3

**Test Plan**

Update some pkgs, verify commit info gets prefilled.

**Checklist**

- [ ] Package was built and tested against unstable

### CODE REVIEW

**Comment by ermo** [comment]
Given that this imports new libraries, doesn't it need to be accompanied by an update to the packaging documentation for ensuring that the user installs pyyaml?

**Comment by joebonrichie** [comment]
Normally i would agree but pyyaml needs to be installed as part of ypkg e.g. go-task bump anyway. I do wonder where the best way to check this would be?

File: common/Hooks/prepare-commit-msg.py [hunk 1]
```diff
@@ -21,6 +23,19 @@
...
+                                            os.path.join(commit_dir, 'package.yml')],
+                                            stdout=subprocess.PIPE)
+        if "version" in recipe_diff_result.stdout.decode('utf-8'):
+            with open(os.path.join(commit_dir, 'package.yml')) as recipe:
+                try:
+                    data = yaml.safe_load(recipe)
+                    version = data['version']
+                    return os.path.basename(commit_dir) + ': Update to ' + str(version)
+                except yaml.YAMLError as e:
+                    print(e)
```
Comment (silkeh) [inline common/Hooks/prepare-commit-msg.py:37]: Should this exception ever occur? I feel like it should never happen, in which case it's fine not to catch the exception:

```suggestion
                    version = yaml.safe_load(recipe)['version']
                    return os.path.basename(commit_dir) + ': Update to ' + str(version)
```

**Review by silkeh — APPROVED** [review]
LGTM, thanks!

**Pull request merged**